integration/templates/next-app-router/package.json

```diff
@@ -9,6 +9,11 @@
     "start": "next start"
   },
   "dependencies": {
+    "@clerk/backend": "file:.yalc/@clerk/backend",
+    "@clerk/clerk-react": "file:.yalc/@clerk/clerk-react",
+    "@clerk/nextjs": "file:.yalc/@clerk/nextjs",
+    "@clerk/shared": "file:.yalc/@clerk/shared",
+    "@clerk/types": "file:.yalc/@clerk/types",
```

Comment:
i think this change will not work for tests running in our CICD and will break the integration tests. As far as i know we don't depend on `yalc` in the integration tests.
cc: @nikosdouvlis